<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>660</width>
     <height>770</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(250, 250, 100, 150);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Gallery">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>590</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 200, 250, 150);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Gallery</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Click">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>590</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 200, 250, 150);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Click</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_different">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>590</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 200, 250, 150);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Different
Filter</string>
   </property>
  </widget>
  <widget class="QLabel" name="Picture_position">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>621</width>
     <height>541</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>601</width>
     <height>521</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
